Automattic/wp-calypso
# Remove Jetpack products feature flags

### Changes proposed in this Pull Request

In this PR, we remove the features flags associated to Jetpack products. The flags in question are set to true in all environments, and are barely used across the codebase.

### Testing instructions

- Review code
- Download the PR and run both Calypso and cloud
- Make sure you see all the Jetpack products and plans in the plans/pricing page
- Check that Backup and Scan work as usual
